changelog/6991.improvement.rst

```diff
@@ -0,0 +1 @@
+Collected files are displayed after any reports from hooks, e.g. the status from ``--lf``.
```

Comment:
Maybe we should drop this change from the backport? (the changes in `src/_pytest/terminal.py`)